packages/react-jsx-highcharts/src/components/PlotBandLine/UsePlotBandLineLifecycle.js

```diff
@@ -25,8 +25,15 @@ export default function usePlotBandLineLifecycle(props, plotType) {
       };
 
       if (plotbandline) axis.removePlotBandOrLine(idRef.current);
+      axis.addPlotBandOrLine(opts, plotType);
       setPlotbandline({
-        object: axis.addPlotBandOrLine(opts, plotType)
+        id: myId,
+        getPlotBandLine: () => {
+          const plotbandlineObject = axis.object.plotLinesAndBands.find(
+            plb => plb.id === myId
+          );
+          return plotbandlineObject;
+        }
```

Comment:
This is technically a breaking change, but then again it did not work at all before.